nototools/noto_lint.py

```diff
@@ -362,7 +362,9 @@ def printable_font_versions(font):
 
 
 def _build_cmap_dict(filename):
-    data = cmap_data.read_cmap_data_file(filename)
+    tooldir = tool_utils.resolve_path('[tools]/nototools')
+    data = cmap_data.read_cmap_data_file(
+        path.join(tooldir, filepath))
```

Comment:
Just a thought. Would it be worth adding a helper method (to tool_utils)  for constructing a full path (that combines resolve_path and path.join above) ?